### PR TITLE
Form Project Review

### PR DESCRIPTION
**Pull request by TashfeenRao**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -14,23 +14,23 @@ def show
   def create
     @user = User.new(user_params)
     if @user.save
-      # success
       flash[:success] = 'User Registered on System!'
-      redirect_to user_url(@user)
+      redirect_to @user
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: According to [that](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png), you have to redirect to `new_user_path`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -14,23 +14,23 @@ def show
...
     @user = User.find(params[:id])
+    # debugger
   end
 
   def update
     @user = User.find(params[:id])
     if @user.update_attributes(user_params)
       flash[:success] = 'User Updated on System!'
-      redirect_to user_url(@user)
+      redirect_to @user
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: You have to redirect to `edit_user_path`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required

Hello @TashfeenRao @konyan ,

Nice improvements :clap: 
However, there are still some errors

1. I got this error when creating a new user
Screenshot from 2020-02-13 18-43-57

2. I got this error when trying to update an user
Screenshot from 2020-02-13 18-47-41

3. You should go through 3 forms, describe [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png), [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) and [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png). We need to see those 3 forms (the 2 first should be commented).
Please, add the 3 first in `edit.html.erb` and `new.html.erb`

4. Please, read the comments left below.

Kindly make the changes and submit a new code review request. Don't create a new one, please, just use the same link with a description added.

Thanks
Mael FOSSO

**Comment by samgaco** [comment]
Hi @TashfeenRao and @konyan :high_brightness: 

Everything is working now and corrected as suggested before, the project is approved :+1: 

-----------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**